sklearn/linear_model/tests/test_least_angle.py

```diff
@@ -686,3 +686,19 @@ def test_lasso_lars_vs_R_implementation():
 
     assert_array_almost_equal(r2, skl_betas2, decimal=12)
     ###########################################################################
+
+
+@pytest.mark.parametrize('copy_X', [True, False])
+def test_lasso_lars_copyX_behaviour(copy_X):
+    """
+    Test that user input regarding copy_X is not being overridden (it was until
+    at least version 0.21)
+
+    """
+    lasso_lars = LassoLarsIC(copy_X=copy_X, precompute=False)
+    X = np.random.normal(0, 1, (100, 5))
```

Comment:
always use a RandomState in tests

Reply:
even if having a consistent seed is not important for this test? the assert results will be correct regardless of the seed. I have added it, if only for style consistency with the other tests in the suite.